msig_contract_deployment/msig_erc20_bootstrap.md

```diff
@@ -0,0 +1,39 @@
+# Multi-sig ERC20 bridge contract bootstrapping
+
+After the creation of ERC20 bridging accounts (`eosio.erc2o` & `eosio.evmin`), we need to bootstrap the bridging contracts:
+
+## Multi-sig 1: Update the latest version of evm_runtime contract & withdraw funds
+
+Action 1: Update the existing evm_runtime contract, adding supports for the following features required for ERC20 bridging:
+  - *call action: this action will allow any account in the EOS side acts as the corresponding account (using deterministric reserverd address) in EVM side*
+  - *admincall action: this action allow ENF & BPs to co-sign to satisfy eosio.evm@active and act on behalf of any EVM account in emergency situations.*
+  - *event mechanisim: this is the mechanisim allowing event generated in the solidity contract in EVM to be sent as bridge message via inline action in native EOS.*
+
+Action 2: withdraw 300 EOS from the eosio.evm's fee balance to enf for covering RAM and as ERC-20 bridge funds 
+
+## Multi-sig 2: ERC-20 contract bootstrapping (depends on Multi-sig 1):
+
+Action 1: Deploy core EVM ERC-20 contract to account `eosio.erc2o`. This contract contains the following actions:
```

Comment:
Technically these are two actions since we need to deploy both the wasm and the abi.